src/Symfony/Bridge/Doctrine/ContainerAwareEventManager.php

```diff
@@ -52,7 +53,7 @@ public function dispatchEvent($eventName, EventArgs $eventArgs = null)
         }
 
         foreach ($this->listeners[$eventName] as $hash => $listener) {
-            $listener->$eventName($eventArgs);
+            $listener->{$this->methods[$hash]}($eventArgs);
```

Comment:
We could also only mark "__invoke" listeners and use `$this->method[$hash] ?? $eventName`

Reply:
good as is to me